ABOUT.md

```diff
@@ -1,9 +1,72 @@
+
+### Free
+
+ * Shogun is **community based** and **non-commercial**.
+ * Licence: Shogun is currently released under GPLv3. We are working towards a BSD compatible repository and all newly added code is required to be BSD3 compliant.
+
+### Good (is what we aim for)
+
+ * Shogun has exclusive **state-of-the art algorithms** (among others: efficient SVM implementations, Multiple Kernel Learning, kernel hypothesis testing and Krylov methods). All of these are supported by a collection of general purpose methods for evaluation, parameter tuning, preprocessing, serialisation and I/O.
+ * Our implementation is **efficient and competitive** and we contribute to the benchmarking project by MLPack and Markus Edel (a framework to compare algorithms from different ML libraries).
+ * Shogun has an **extensive testing infrastructure**, making it reliable on dozens of OS setups.
+
+## <a name="useshoguntolearn"></a> Using Shogun to Learn
+
+Shogun is a community learning platform in every sense: **anyone can use the toolbox to learn about ML** and apply it to solve problems.
+
+ * Many **scientists** use Shogun as a tool for their research and it is [cited many hundred times on Google scholar](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=).
+ * We offer the Shogun cloud for **educational** purposes at universities and workshops (e.g. University College London GradSchool Introduction to Machine Learning).
+
+Even more important, **everyone can gain skills by helping us to build Shogun**. We highly value our contributors and want to foster and develop them.
+
+ * We have mentored 33 **Google Summer of Code student projects** since 2011 (and counting).
+ * Many of our alumni students took their acquired skills to industry and we are proud to be part of their education.
+
+## <a name="shogunhistory"></a> Shogun History
+
+Shogun got initiated in 1999 by [Soeren Sonnenburg](http://sonnenburgs.de/soeren) and [Gunnar Raetsch](http://www.raetschlab.org/) (that's where the name *ShoGun* originates from). The original focus was on large-scale kernel methods and bioinformatics. Since then, Shogun has continuously been used for scientific research (a list of papers citing Shogun can be found [here](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=)) and massively expanded.
+
+Shogun’s features now span the whole space of Machine Learning methods: it includes many classic methods in classification, regression, dimensionality reduction and clustering, but also advanced algorithm classes such as metric, multi-task, structured output and online learning, as well as feature hashing, ensemble methods and optimization (to name just a few). We aim to keep up with exclusive state-of-the art algorithms and have built a wealth of efficient SVM implementations, Multiple Kernel Learning, kernel hypothesis testing and Krylov methods, etc.
```

Comment:
as I said before - please tell me more about the history so this can actually be a history. It doesn't need to be integrated into the features because everything that needs to be in features is already there